conanfile.py

```diff
@@ -80,29 +79,5 @@ def build(self):
         if(self.options.buildtests):
             self.cmake.test()
 
-    # def package(self):
-    #    self.copy("*.h", dst="include", src="NovelRT/include")
-    #     # self.copy("*.h", dst="include/NovelRT", src="NovelRT/include/NovelRT")
-    #     # self.copy("*.h", dst="include/NovelRT.Interop", src="NovelRT/include/NovelRT.Interop")
-    #     if sys.platform.startswith('win32'):
-    #         self.copy("*NovelRT.lib", dst="lib", keep_path=False)
-    #         self.copy("*NovelRT.Interop.lib", dst="lib", keep_path=False)
-    #         self.copy("*NovelRT.dll", dst="bin", keep_path=False)
-    #         self.copy("*NovelRT.Interop.dll", dst="bin", keep_path=False)
-    #         self.copy("*bz2.dll", dst="bin", keep_path=False)
-    #         self.copy("*FLAC.dll", dst="bin", keep_path=False)
-    #         self.copy("*FLAC++.dll", dst="bin", keep_path=False)
-    #         self.copy("*fmt.dll", dst="bin", keep_path=False)
-    #         self.copy("*freetype.dll", dst="bin", keep_path=False)
-    #         self.copy("*glfw3.dll", dst="bin", keep_path=False)
-    #         self.copy("*ogg.dll", dst="bin", keep_path=False)
-    #         self.copy("*OpenAL32.dll", dst="bin", keep_path=False)
-    #         self.copy("*opus.dll", dst="bin", keep_path=False)
-    #         self.copy("*sndfile.dll", dst="bin", keep_path=False)
-    #         self.copy("*vorbis.dll", dst="bin", keep_path=False)
-    #         self.copy("*vorbisenc.dll", dst="bin", keep_path=False)
-    #         self.copy("*vorbisfile.dll", dst="bin", keep_path=False)
-
```

Comment:
Unneeded - this is only if we're distributing, which we're not with this particular conanfile.